Source/spells.h

```diff
@@ -9,9 +9,17 @@
 
 namespace devilution {
 
+enum class SpellFailReason : uint8_t {
+	Success = 0,
+	None = 0,

```

Comment:
Why have 2 names for the same result? This seems like a code smell to me.

Reply:
Yes I also hesitated changed it back and forth. checking the result of function calls feels more naturally if `checkSpell == success` instead of `checkSpell == none` but on other places `none` looks more natural.